doc-examples/jdbc-example-records-java/src/test/java/example/BookRepositorySpec.java

```diff
@@ -37,7 +37,7 @@ void testAnnotationMetadata() {
                 .orElse(null);
 
         assertEquals( // <4>
-                "SELECT book_.`id`,book_.`date_created`,book_.`title`,book_.`pages` FROM `book` book_ WHERE (book_.`title` = ?)",
+                "SELECT book_.* FROM `book` book_ WHERE (book_.`title` = ?)",
```

Comment:
Generally `SELECT *` can retrieve more data than is necessary, is there a reason for this change?

Reply:
It looked like a good idea, but if we assume that the table can have columns that aren't mapped then it's not.

Reply:
Yeah we assume that, IMO with Micronaut Data JDBC it will likely be desirable having multiple entities map to the same table